LICENSE.md

```diff
@@ -1,6 +1,6 @@
 The MIT License (MIT)
 
-Copyright (c) 2016 Johannes Köster, the Rust-Bio team.
+Copyright (c) 2016 Johannes Köster, the Rust-Bio team, Google Inc.
```

Comment:
So, is this official google work? Are you part of google? If so, I'm excited to hear that google is interested in this project!!